.travis.yml

```diff
@@ -1,26 +1,59 @@
 language: scala
 sudo: false
-scala:
-- 2.12.6
-- 2.11.12
+
 install:
 - rvm use 2.4.4 --install --fuzzy
 - gem update --system
 - gem install sass
 - gem install jekyll -v 3.8.2
-jdk:
-- oraclejdk8
+
+stages:
+- name: test
+- name: release
+  if: (branch = master AND type = push) OR (tag IS present)
+
+matrix:
+  include:
+  - jdk: oraclejdk8
+    scala: 2.11.12
+    env: TEST="scalafmt"
+    script: sbt ++$TRAVIS_SCALA_VERSION scalafmtCheck test:scalafmtCheck scalafmtSbtCheck
+  - jdk: oraclejdk8
+    scala: 2.11.12
+    env: TEST="scalafmt"
+    script: sbt ++$TRAVIS_SCALA_VERSION scalafmtCheck test:scalafmtCheck scalafmtSbtCheck
+  - jdk: oraclejdk8
+    scala: 2.12.6
+    env: TEST="test"
+    script: sbt ++$TRAVIS_SCALA_VERSION coverage test coverageReport microsite/makeMicrosite && bash <(curl -s https://codecov.io/bash)
+  - jdk: oraclejdk8
+    scala: 2.12.6
+    env: TEST="test"
+    script: sbt ++$TRAVIS_SCALA_VERSION coverage test coverageReport microsite/makeMicrosite && bash <(curl -s https://codecov.io/bash)
+  - stage: release
+    jdk: oraclejdk8
+    scala: 2.11.12
+    script: sbt ci-release
```

Comment:
You need `++$TRAVIS_SCALA_VERSION` to be part of the ci-release

Reply:
oh, thanks!